src/OMSimulatorLib/ComponentTable.h

```diff
@@ -94,6 +96,7 @@ namespace oms
     std::unordered_map<unsigned int /*result file var ID*/, unsigned int /*allVariables ID*/> resultFileMapping;
     double time;
     double storedTime;
+    std::vector<ComRef> filteredSignals;  ///< filtered signals set via oms_addSignalsToResults()
```

Comment:
Isn't this sort of the same as `std::unordered_map<ComRef, bool> exportSeries;`?

Reply:
@lochel  this should be fixed with this https://github.com/OpenModelica/OMSimulator/pull/915/commits/ae0fab74593c4754886d1fa6bf51e92013f1f19b